stdlib/asm/crypto/stark/random_coin.masm

```diff
@@ -75,119 +73,103 @@ end
 #! equal to 8.
 #! The ouput of this procedure is the capacity portion of the state after applying `hperm`.
 #!
-#! Input: [log(trace_length), num_queries, log(blowup), grinding, ...]
+#! Input: [log(trace_length), num_queries, blowup, grinding, ...]
 #! Output: [C]
-#! Cycles: 97
+#! Cycles: 104
 export.init_seed
 
-    # Compute parameters and save them to memory
-    # ---------------------------------------------------------------------------------------------
-
-    # Save `log(trace_length)`, `num_queries`, and `grinding_factor` to memory
+    # Save the parameters in memory for later use
     dup exec.constants::trace_length_log_ptr mem_store
     dup.1 exec.constants::number_queries_ptr mem_store
     dup.3 exec.constants::grinding_factor_ptr mem_store
-    #=> [log(trace_length), num_queries, log(blowup), grinding, ...] (9 cycles)
```

Comment:
This reverted documentation changes introduced by https://github.com/0xPolygonMiden/miden-vm/pull/973 , was that intentional?

I was relying on that to understand the stack and implement https://github.com/0xPolygonMiden/miden-vm/pull/960

Reply:
I probably removed it by mistake as I don't see now a reason for removing it

Reply:
It was probably me who removed it by mistake because I screwed up commit history at some point.